packages/router/src/recognize.ts

```diff
@@ -407,6 +425,9 @@ export class Recognizer {
               route,
               getResolve(route),
             );
+            const i = getInherited(snapshot, parentRoute, this.paramsInheritanceStrategy);
```

Comment:
nit: I'd propose using a full name:
```suggestion
            const inherited = getInherited(snapshot, parentRoute, this.paramsInheritanceStrategy);
```
(we may need to pass it somewhere or use in a different part of the function later, so it'd be easier to refactor later)